caluma/caluma_form/structure.py

```diff
@@ -91,8 +91,8 @@ def value(self):
                 for row in self.children()
             ]
 
-        elif self.question.type == Question.TYPE_FILE:
-            return self.answer.file
+        elif self.question.type == Question.TYPE_FILES:
+            return [f.name for f in self.answer.files.all()]
```

Comment:
Now we return the `name` of the file while before we returned the entire `file`. Is that the same in this context or is that an intended change?